src/app/web-app.component.ts

```diff
@@ -113,6 +126,141 @@ export class WebAppComponent implements OnInit {
         verticalPosition: 'top'
       });
     });
-  }
 
+    // Keyboard Shortcuts Listener
+    this.unlisten = this.keyboardShortcuts.listen(
+      {
+        // Navigation Page
+        'Control.Shift.n': (event: KeyboardEvent): void => {
+          this.router.navigate(['navigation']);
+          event.preventDefault();
+        },
+
+        // Checker Inbox & Pending Tasks
+        'Control.Shift.i': (event: KeyboardEvent): void => {
+          // this.router.navigate(['navigation']);
+          console.log('Not Implemented Yet!!');
+          event.preventDefault();
+        },
+
+        // Collection Sheet
+        'Control.Shift.o': (event: KeyboardEvent): void => {
+          // this.router.navigate(['navigation']);
+          console.log('Not Implemented Yet!!');
+          event.preventDefault();
+        },
+
+        // Create Client
+        'Control.Shift.c': (event: KeyboardEvent): void => {
+          // this.router.navigate(['navigation']);
+          console.log('Not Implemented Yet!!');
+          event.preventDefault();
+        },
+
+        // Create Group
+        'Control.Shift.g': (event: KeyboardEvent): void => {
+          // this.router.navigate(['navigation']);
+          console.log('Not Implemented Yet!!');
+          event.preventDefault();
+        },
+
+        // Create Center
+        'Control.Shift.q': (event: KeyboardEvent): void => {
+          // this.router.navigate(['navigation']);
+          console.log('Not Implemented Yet!!');
+          event.preventDefault();
+        },
+
+        // Frequent Postings
+        'Control.Shift.f': (event: KeyboardEvent): void => {
+          this.router.navigate(['accounting/journal-entries/frequent-postings']);
+          event.preventDefault();
+        },
+
+        // Closure Entries
```

Comment:
Accounting closure entries. This has been implemented. Recheck and update accordingly.